cmd/notify-mailer/main.go

```diff
@@ -368,51 +442,43 @@ Examples:
 
   notify-mailer -config test/config/notify-mailer.json -body
     cmd/notify-mailer/testdata/test_msg_body.txt -from hello@goodbye.com
-    -recipientList cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
-    -sleep 10s -dryRun=false
+    -recipient-list cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
+    -sleep 10s -dry-run=false
 
   Do the same, but only to example@example.com:
 
   notify-mailer -config test/config/notify-mailer.json
     -body cmd/notify-mailer/testdata/test_msg_body.txt -from hello@goodbye.com
-    -recipientList cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
+    -recipient-list cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
     -start example@example.com -end example@example.comX
 
   Send the message starting with example@example.com and emailing every address that's
 	alphabetically higher:
 
   notify-mailer -config test/config/notify-mailer.json 
     -body cmd/notify-mailer/testdata/test_msg_body.txt -from hello@goodbye.com 
-    -recipientList cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
+    -recipient-list cmd/notify-mailer/testdata/test_msg_recipients.csv -subject "Hello!"
     -start example@example.com
 
 Required arguments:
 - body
 - config
 - from
 - subject
-- recipientList`
+- recipient-list-csv OR recipient-list-tsv`
```

Comment:
Why not have a `recipient-list` flag, and a separate `format` or `delimeter` flag?